### ISSUE TITLE
Putting text on ButtonFloat

### ISSUE THREAD

**Issue by paragones** [navasmdc/MaterialDesignLibrary#306]
Hi, 

Is it possible to put a text on ButtonFloat properties into the xml file?

Here's my code. I don't really need an image there. 

    <com.gc.materialdesign.views.ButtonFloat
            android:id="@+id/button_delete_hours"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:minHeight="1dp"
            android:minWidth="1dp"
            android:background="@color/orange_main"
            android:text="X"
            android:textColor="#9e9e9e"
            android:textSize="15sp"
            android:layout_alignParentRight="true"
            android:layout_marginRight="16dp"
            android:layout_marginTop="27dp"
            />

**Comment by mkorniluk** [comment]
No, it's not possible. ButtonFloat is a RelativeLayout with ImageView inside.
